samcli/lib/replace_uri/replace_uri.py

```diff
@@ -0,0 +1,94 @@
+from samcli.lib.utils.packagetype import ZIP
+
+
+def _replace_local_codeuri(template):
```

Comment:
despite what the docstring explains, I feel the name of the function is a bit ambiguous. Can it be sth like `_replace_local_path_with_fake_s3_uri` that is less ambiguous about what it exactly does?

Reply:
Since all of the code in this file was copied from sam_template_validator, I wanted to keep it as consistent and similar to what it originally was. If you still think that updating the function name would improve its readability, I can change it.